src/lib/y2partitioner/widgets/bcache_device_description.rb

```diff
@@ -63,9 +63,18 @@ def caching_device
       end
 
       def backing_device
-        return "" unless device.blk_device
+        return "" if device.is?(:flash_bcache) || device.backing_device.nil?
```

Comment:
what about marking somehow flash only volume? As bcache can have two possibilities. bcache_cset that is not associated and flash_only volume. So what about using something like `_("(flash only volume)")` for them?